features/step_definitions/testdebt_generic_test_data_step.rb

```diff
@@ -0,0 +1,64 @@
+Then(/^I override deed with "([^"]*)"$/) do |params|
+  # This will work for both steps below:
+  # => And I override deed with "middle_name:Synth:2"
+  # and
+  # => And I override deed with "middle_name:Synth"
+  # The 3rd value is optional.
+
+  # Sets opt = 1, if no bor parameter is passed in
+  opt ||= 1
+  # Splits the params string
+  split_params = params.split(':')
+  var = split_params[0]
+  value = split_params[1]
+
+  # Checks if Borrower number was passed in
+  if split_params[2] != nil
+    opt = split_params[2]
+  end
+
+  case var
```

Comment:
Given how similar the individual parts of this case statement are I'm wondering if there's a more efficient way this can be done? Whether we can just call @deed.<whatever> = value and pass the var into the statement like that?